db/schema.rb

```diff
@@ -152,7 +161,7 @@
   create_table "users", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb3", force: :cascade do |t|
     t.string "name"
     t.string "full_name"
-    t.string "password_digest", default: "$2a$12$6ONW8hZL3q2oOABLut6NOO4cDU9uULpu0If1OVVsfTgaURpk5SCAW"
+    t.string "password_digest", default: "$2a$12$Hr0rGnOdIXz3BpwzQhjmlulYdYc3B7XkXhdzi581VYbtBxOijOYWm"
```

Comment:
Ok to have secrets in plain view? Just 2x check

Reply:
This is the default password for admins (for them to log in the first time) stored in hash. I think this is fine. But, if you want to, I can add a credentials file to store this and other sensitive hash values across the whole repo.